src/tilt_ble/parser.py

```diff
@@ -83,9 +80,9 @@ def _start_update(self, service_info: BluetoothServiceInfo) -> None:
             native_value=temperature,
         )
         self.update_sensor(
```

Comment:
You could use `self.update_predefined_sensor` here instead to save some lines

Reply:
I thought about that. All the other examples I saw used constants defined in the [sensor-state-data library](https://github.com/Bluetooth-Devices/sensor-state-data/blob/main/src/sensor_state_data/library.py). Is it expected that I add to that library first, or just directly construct a `BaseSensorDescription`?

Reply:
You would need to add it to `SensorLibrary` to use that